administrator/components/com_menus/tmpl/item/edit_modules.php

```diff
@@ -98,37 +102,21 @@ class="btn btn-link module-edit-link"
 					data-module-id="<?php echo $module->id; ?>">
 					<?php echo $this->escape($module->title); ?></button>
 			</td>
+			<td id="menus-<?php echo $module->id; ?>" style="width:15%" >
+				<div class="switcher has-success" role="switch" >
+					<input type="radio" id="jform_showModule0" name=<?php echo "jform[".$modules[$module->title]."]"?> value="0" class="valid form-control-success" aria-invalid="false" method="post">		
```

Comment:
Why did you used $module->title ?

Reply:
I have created an associative array where module title is the key and the module id is the value 
To fetch the key value I'm using $module->title